weave-js/src/components/PagePanelComponents/Home/Browse3/typeViews/CustomWeaveTypeDispatcher.tsx

```diff
@@ -28,6 +29,12 @@ const customWeaveTypeRegistry: {
   'PIL.Image.Image': {
     component: PILImageImage,
   },
+  'openai._legacy_response.HttpxBinaryResponseContent': {
```

Comment:
similar to other PR, i really don't think `HttpxBinaryResponseContent` is just for audio